code/TemplateStudioForWinUICs/Templates/_comp/MT/Project.SplitView/Views/ShellPage.xaml.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using Microsoft.UI.Xaml.Controls;
 using Microsoft.UI.Xaml.Input;
+using Param_RootNamespace.Helpers;
```

Comment:
Please sort alphabetically. Helpers should come between Contracts and ViewModels.

Reply:
There's a postaction that sorts all namespaces in the generated code. 
If we want to ensure they're presorted in the templates too, we should add a test/check for this as part of the minimal tests.

Reply:
The test is not a priority right now for 5.1 but we should make the change here.